documentation/analytics/usage.md

```diff
@@ -120,6 +120,11 @@ Once you've exported the sprint and issue data from GitHub, you can start calcul
 poetry run analytics calculate sprint_burndown --sprint-file data/sprint-data.json --issue-file data/issue-data.json --sprint @current --unit points --show-results
 ```
 
+If on a Windows machine: 
```

Comment:
Instead of specifying a different command for windows, might be worth just tweaking the guidance below to wrap `@current` in quotes since it also works with quotes on Mac/Linux

Reply:
Good call. Done.